carbon-design-system/carbon
# remove extra packages path from icons readme

#### Changelog

**Changed**

- `packages/icons/README.md` to remove duplicate `packages` string that appears in relative path link to `packages/icon-helpers`.

#### Testing / Reviewing

1. Navigate to [`packages/icons/README.md`](https://github.com/carbon-design-system/carbon/blob/master/packages/icons/README.md#vanilla)
1. Click `icon-helpers` hyper-link
1. Verify that the `icon-helpers` package is actually opened
